src/alpaca_eval/leaderboards/data_AlpacaEval_2/weighted_alpaca_eval_gpt4_turbo_leaderboard.csv

```diff
@@ -1,10 +1,10 @@
 ,win_rate,standard_error,n_wins,n_wins_base,n_draws,n_total,discrete_win_rate,mode,avg_length,length_controlled_winrate,lc_standard_error
```

Comment:
why remove all those models? Maybe you haven't rebased on main?

Reply:
Hi, I just corrected the file.